addons/binding/org.openhab.binding.max/ESH-INF/thing/bridge.xml

```diff
@@ -15,37 +15,68 @@
 		</channels>
 
 		<config-description>
-			<parameter name="ipAddress" type="text" required="true">
+			<parameter-group name="identification">
+				<label>Identification</label>
+				<description>Hardware &amp; location identification</description>
+				<context>thing</context>
```

Comment:
Imho context=thing means that the content of the string is a Thing UID - it is meant as a hint to UIs to allow rendering of appropriate selection boxes.